src/types/machines/onboarding/index.ts

```diff
@@ -8,6 +8,15 @@ import {OnboardingStackParamsList} from '../../navigation';
 import {IUser} from '../../user';
 import {MappedCredential} from '../getPIDCredentialMachine';
 import {PIDSecurityModel} from '../../../services/storageService';
+import {CredentialPayload, ProofFormat} from '@veramo/core';
+import {SupportedDidMethodEnum} from '../../did';
+
+export type OnboardingCredentialData = {
+  didMethod: SupportedDidMethodEnum;
+  didOptions?: any;
```

Comment:
Is this one needed? Since IDIDOptions is deprecated.

Reply:
i took it from whatever we were doing before, so for now i would say, address it later if needed